general/releases.rst

```diff
@@ -20,9 +20,13 @@ old, upcoming or future release.
 +---------------+--------------+
 | Version       | Release date |
 +===============+==============+
+| OP-TEE 3.15.0 | 15/Oct/21    |
++---------------+--------------+
+| OP-TEE 3.14.0 | 16/July/21   |
```

Comment:
At this point should we consider adopting the [TeX versioning scheme](https://en.wikipedia.org/wiki/TeX#TeX82)? (just kidding :wink:)

Reply:
Haha, what an opportunity! :-)